routes/controllers/auth.controller.js

```diff
@@ -0,0 +1,41 @@
+const { OAuth2Client } = require("google-auth-library");
+const User = require("../../models/User");
+
+exports.googleAuth = async (req, res, next) => {
+  const client = new OAuth2Client(process.env.EXPO_CLIENT_ID);
+  const { id_token } = req.body;
+  let decoded;
```

Comment:
```suggestion
  const { id_token } = req.body;
  
  let decoded;
```

const와 let 사이를 띄워주면 좋습니다!